packages/semantic-model/src/convert.ts

```diff
@@ -162,9 +162,10 @@ function convertClass(
   symbol: apiJson.ClassSymbol
 ): model.UI5Class {
   const base = convertSymbol(libName, symbol);
-  const classs: model.UI5Class = {
+  const classes: model.UI5Class = {
```

Comment:
It's one class... I couldn't call it `class` because it's a reserved word so I added an extra `s`

Reply:
ah I thought its a typo and created a new typo, I will rename to `clazz`